commands/pr-id.sh

```diff
@@ -18,7 +21,16 @@ pr-id() {
         }
       }
     }
-  ' | jq -r '.data.repository.pullRequests.edges[0].node[]' 
+  ' | jq -r '(.data.repository.pullRequests.edges[0].node // {})[]')"
+
+  (( "$?" )) && return 1
```

Comment:
I personally insist that `pr-id` be simply `gh` and `jq` without error handling.

Reply:
[IMO] I want to show error messages when if foobar script execute by itself.

Reply:
I think so should not error handling for vimscript side.